app/models/Dataset.scala

```diff
@@ -38,6 +39,55 @@ case class Dataset(
   def isDefault:Boolean = status == DatasetStatus.DEFAULT.toString
   def isTRIAL:Boolean = status == DatasetStatus.TRIAL.toString
   def inSpace:Boolean = spaces.size > 0
+
+  /**
+    * Caps a list at 'max'  
+    * then turns it's ID's into resolvable URLs of that 'apiRoute' type
+    * end with appending "..." to the List, to signify that it was abridged 
+    *
+    * todo: issue 354 to the max configurable
+    */
+  def cap_api_list (l: List[UUID], max: Int, URLb: String, apiRoute: String) : List[String] = {  
+      if (l.length <= max)  {
+        return l.map(f => URLb + apiRoute + f) 
+      } else {
+         val cl = l.take(max)
+         val r : List[String] = cl.map(f => URLb + apiRoute + f) 
+         return r.::("...").reverse 
+      }
+   } 
+
+  /**
+    * return Dataset as JsValue in jsonld format
+    */
+  def to_jsonld(url: String) : JsValue = { 
+     val so = JsObject(Seq("@vocab" -> JsString("https://schema.org/")))
+     val URLb = url.replaceAll("/$", "") 
+     var pic_id = thumbnail_id.getOrElse("")
+     if (pic_id != "") {
+        pic_id = URLb + pic_id 
+     } else ""
+     val datasetLD = Json.obj(
+           "context" -> so,
```

Comment:
I was playing around the json-ld playground and realized that this needs to be `@context` to be properly parsed.

Reply:
replaced and tested